STYLE_GUIDE.md

```diff
@@ -0,0 +1,97 @@
+# panther-analysis Style Guide
+
+## Metadata Best Practices
+
+### RuleID, Filename and DisplayName
+
+- RuleID, Filename and DisplayName should be similar enough to eachother to be able to easily identify a filename by it's RuleID or vice versa.
```

Comment:
sugg: 
- RuleID, Filename, and DisplayName should all be similar to one another. A good litmus test is: If you have the RuleID, would you be able to identify the related Python file (its Filename), and vice versa?